src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  margin: auto;
+  flex-wrap: wrap;
+  gap: $base-gap;
+  width: ($square-size * 7) + ($base-gap * 6);
+  padding: 10px;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    height: $square-size;
+    width: $square-size;
+    background-color: $square-color;
+    border: 1px solid black;
+    transition: 500ms;
+
+    &::before {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      margin: auto;
+    }
```

Comment:
`::before` pseudo-element is used here, but it doesn't have any content, so it doesn't have any effect on the page. If you intended to use it for centering the content, you can achieve this by applying `justify-content: center;` and `align-items: center;` to the `.calendar__day` class itself.